### ISSUE TITLE
"prescribes" needs a definition

### ISSUE THREAD

**Issue by mbrochhausen** [OOSTT/OOSTT#12]

**Comment by jonathanbona** [comment]
A range would be nice too. 

Is it safe to assume the intended meaning here is closer to sense 1 below than to sense 2?

> transitive verb
> 1 a : to lay down as a guide, direction, or rule of action : ordain
> b : to specify with authority
> 2 : to designate or order the use of as a remedy prescribed a painkiller a prescribed burn to restore natural forest conditions
- https://www.merriam-webster.com/dictionary/prescribe

**Comment by mbrochhausen** [comment]
Yes, when I talked about defining, in my mind it would be clear that a domain and range would be nice.
I think the intended meaning is 1a (although how much authority is involved seems to be a tricky question).
